src/ufuncs.c

```diff
@@ -1711,10 +1720,41 @@ static void to_wkb_func(char **args, npy_intp *dimensions,
             Py_INCREF(Py_None);
             *out = Py_None;
         } else {
+            /* WKB Does not allow empty points.
+             * We check for that and patch the case of a 'simple' POINT EMPTY.
+             * Collections with POINT EMPTY in it will raise an exception
```

Comment:
is this comment still up to date? Because you now also handle MultiPoint / GeometryCollection with empty points?